ibm/service/power/resource_ibm_pi_volume_clone.go

```diff
@@ -53,6 +59,7 @@ func ResourceIBMPIVolumeClone() *schema.Resource {
 				Type:        schema.TypeBool,
 			},
 			Arg_UserTags: {
+				Computed:    true,
```

Comment:
`Arg_UserTags` is a ForceNew attribute, does not need Computed or customize diff since it is not read (CRN is not returned)

Reply:
On second thought, we can probably leave this in.